New-LoopAction.ps1

```diff
@@ -123,8 +123,16 @@ function New-LoopAction {
                         }
                     }
                     . $ScriptBlock
+                    switch (. $ExitCondition) {
+                        $false {
+                            $ExitConditionResult = $false
+                        }
+                        Default {
+                            $ExitConditionResult = $true
+                        }
+                    }
```

Comment:
Can we simply assign $ExitConditionResult directly?

```suggestion
                    $ExitConditionResult = . $ExitCondition
```

Reply:
Absolutely